lxc/copy.go

```diff
@@ -35,6 +36,7 @@ func (c *copyCmd) flags() {
 	gnuflag.Var(&c.profArgs, "p", i18n.G("Profile to apply to the new container"))
 	gnuflag.BoolVar(&c.ephem, "ephemeral", false, i18n.G("Ephemeral container"))
 	gnuflag.BoolVar(&c.ephem, "e", false, i18n.G("Ephemeral container"))
+	gnuflag.BoolVar(&c.containerOnly, "container-only", false, i18n.G("Copy container without snapshots"))
```

Comment:
For consistency in sentence structure with the others, that ought to be "Copy the container without its snapshots".